integration-tests/src/tests/client/features/delegate_action.rs

```diff
@@ -137,7 +141,14 @@ fn check_meta_tx_execution(
         .nonce;
     let user_pubk = match sender.get_account_type() {
         AccountType::NearImplicitAccount => PublicKey::from_near_implicit_account(&sender).unwrap(),
-        AccountType::EthImplicitAccount => PublicKey::from_seed(KeyType::ED25519, sender.as_ref()),
+        AccountType::EthImplicitAccount => {
+            if checked_feature!("stable", EthImplicitAccounts, protocol_version) {
+                // ETH-implicit accounts must not have access key.
+                panic!("No access keys");
```

Comment:
nit: Can you change the panic message to the comment above and remove the comment?